apps/nextjs-website/src/lib/types/guidesData.ts

```diff
@@ -1,4 +1,4 @@
 import { Page } from '@/lib/types/page';
 import { GuidesPageProps } from '@/pages/[productSlug]/guides';
 
-export type GuidesData = Page<GuidesPageProps>;
+export type GuidesData = Page<Omit<GuidesPageProps, 'products'>>;
```

Comment:
The property `products` does not exist
```suggestion
export type GuidesData = Page<GuidesPageProps>;
```

Reply:
Ignore this suggestion. I didn't see that `LayoutProps` has the `products` property.